### DISCUSSION TITLE
introducing myself to the MDAnalysis team

### DISCUSSION THREAD

**Discussion by anivenk25** [MDAnalysis/mdanalysis#4412] [Introductions]
My name is Anirudh V, and I am writing to let you know that I am interested in participating in the Google Summer of Code (GSoC) program with MDAnalysis.

I am currently a NIT Rourkela student pursuing a degree in Computer Science. Through my coursework and independent projects, I have developed a strong foundation in Machine Learning and have completed hackathon-winning DDI and patient diagnosis models, which align well with the requirements of MDAnalysis.

I have been following MDAnalysis for quite some time and am impressed by the impactful contributions it has made to the field of computational molecular dynamics. The prospect of contributing to such a dynamic and innovative project excites me, and I am eager to apply my skills and enthusiasm to enhance MDAnalysis further.

I would like for someone to show me the ropes on how to contribute since I am new to the codebase of the organization.

**Comment by HeetVekariya** [comment]
- Hello @anivenk25, welcome to MDAnalysis.
- This is the comment i got from maintainers when started contributing in MDAnalysis, hope it will help you to get started.


> MDAnalysis is scientific software so it helps to learn more about what it does: see
> https://www.mdanalysis.org/pages/learning_MDAnalysis/ which contains links and videos.
>
> We then recommend you install it and work through the QuickStart guide
> https://userguide.mdanalysis.org/stable/examples/quickstart.html
>
> Our Contributing guide tells you how to get started as a developer https://userguide.mdanalysis.org/stable/contributing.html .

Note: When you try to set up MDAnalysis, please visit this discussion if you got any errors: https://github.com/MDAnalysis/mdanalysis/discussions/4358

**Reply by anivenk25** [reply]
Thanks, @HeetVekariya will go through the guides and get up to speed.
